docs/conferences/ismir2019.md

```diff
@@ -0,0 +1,124 @@
+---
+title: ISMIR 2019
+---
+
+## [Conferences](/conferences) / ISMIR 2019
+
+[Full Proceedings](http://archives.ismir.net/ismir2019/2019_Proceedings_ISMIR.pdf)
```

Comment:
This is new in the 2019 file compared to others. Is this a good place to put it?